LoopMusic.xcodeproj/project.pbxproj

```diff
@@ -240,6 +242,7 @@
 				39B5B31E23B97A1C0091D2E1 /* Notifications.swift */,
 				3973B5F524BAB04100883011 /* NumberUtils.swift */,
 				39DFF47F24B80135003D8E7B /* Unloadable.swift */,
+				9252279B24C419DC00E25C15 /* FontUtils.swift */,
```

Comment:
Xcode is annoying and doesn't automatically sort files alphabetically. Keep files sorted unless there's a specific reason to not do so.

Reply:
Resolved in a31fa0e